specification/security/resource-manager/Microsoft.Security/stable/2021-06-01/assessmentMetadata.json

```diff
@@ -719,9 +719,8 @@
               "Network Sniffing",
               "Non-Application Layer Protocol",
               "Non-Standard Port",
-              "Obfuscated Files or Information",
               "Obtain Capabilities",
-              "Obuscated Files or Information",
+              "Obfuscated Files or Information",
```

Comment:
Please remove the typo from whitelist https://github.com/Azure/azure-rest-api-specs/blob/main/custom-words.txt

Reply:
Fixed.

Reply:
Hi,
Please approve the change.

Reply:
@jianyexi